src/prototypes.h

```diff
@@ -279,9 +279,9 @@ bool unc_homedir(std::string &home);
  * @return the next tabstop column
  */
 static_inline
-size_t calc_next_tab_column(int col, int tabsize)
+size_t calc_next_tab_column(size_t col, size_t tabsize)
```

Comment:
Thanks for the questions.
1.
The function ```calc_next_tab_column``` is always use with ```cpd.settings[UO_input_tab_size].u```,
which value is always >=1 and <=32:
```
   unc_add_option("input_tab_size", UO_input_tab_size, AT_UNUM,
                  "The original size of tabs in the input. Default=8", "", 1, 32);
```
2.
This is used if ```--frag       : Code fragment, assume the first line is indented correctly.```
At this time, ```cpd.frag_cols``` gets a value > 0.
All the next column value(s) are OK. This code is older as I. Never got a problem.